### PR TITLE
Remove more warnings from tests

### PR DESCRIPTION
**Pull request by dstansby**
Some more stuff pulled out of #3648.

### CODE REVIEW

File: sunpy/map/tests/test_compositemap.py [hunk 1]
```diff
@@ -7,10 +7,13 @@
 
 import sunpy.data.test
 import sunpy.map
-from sunpy.tests.helpers import figure_test
+from sunpy.tests.helpers import figure_test, fix_map_wcs
```
Comment (nabobalis) [inline sunpy/map/tests/test_compositemap.py:10]: This is unused?

File: sunpy/net/helio/tests/test_helio.py [hunk 2]
```diff
@@ -12,6 +12,8 @@
     wsdl_retriever,
 )
 
+pytestmark = pytest.mark.filterwarnings('ignore:Unverified HTTPS request is being made')
```
Comment (Cadair) [inline sunpy/net/helio/tests/test_helio.py:15]: why are we making unverified https requests?!

Comment (dstansby) [reply sunpy/net/helio/tests/test_helio.py:15]: 🤷 is this something we should fix instead of ignoring?

Comment (Cadair) [reply sunpy/net/helio/tests/test_helio.py:15]: I think we should understand it at least. Could be cret issues with the helio servers I guess?

Comment (nabobalis) [reply sunpy/net/helio/tests/test_helio.py:15]: We had to disable the SSL check for helio.

Comment (dstansby) [reply sunpy/net/helio/tests/test_helio.py:15]: I'll open an issue with the traceback

Comment (Cadair) [reply sunpy/net/helio/tests/test_helio.py:15]: > We had to disable the SSL check for helio.

We did? I thought it didn't actually fix it?

Comment (nabobalis) [reply sunpy/net/helio/tests/test_helio.py:15]: That was for some instr data.

**Pull request merged**